app/core/api/api-clients.coffee

```diff
@@ -0,0 +1,23 @@
+fetchJson = require './fetch-json'
+
+module.exports = {
+
+  url: (clientID) -> "db/api-clients/#{clientID}/new-secret"
```

Comment:
Let's rename this to be more specific than `url`, such as `urlNewSecret`.  Or, use string directly in `createSecret` to follow the model of other methods here.